NRasterizer.Tests/TestFonts.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NRasterizer.Tests
+{
+    public static class TestFontsPaths
+    {
+        private static readonly string rootFolder;
+
+        static TestFontsPaths()
+        {
+            rootFolder =  Path.GetDirectoryName( new Uri(typeof(TestFonts).Assembly.Location).LocalPath);
+        }
+
+        public static string OpenSans_Regular => $"{rootFolder}/TestFonts/OpenSans-Regular.ttf";
+        public static string SegoeUI => $"{rootFolder}/TestFonts/segoeui.ttf";
+        public static string CompositeMS => $"{rootFolder}/TestFonts/CompositeMS.ttf";
+
+        public static IEnumerable<string> AllFonts
+        {
+            get
+            {
+                yield return OpenSans_Regular;
+                yield return SegoeUI;
+                yield return CompositeMS;
+            }
+        }
+    }
```

Comment:
My first pass didn't have the `AllFonts` property and i was pulling them in individually, i'll drop the individual properties and switch out to simpler version you suggested.